user_events.py

```diff
@@ -409,9 +417,13 @@ def user_an_event_edit(id):
                     # unlink between subevent and super event.
                     if 'id' in old_sub_event:
                         update_super_event_id(old_sub_event['id'], '')
-                    else:
+                    elif 'eventid' in old_sub_event:
                         update_super_event_id_2(old_sub_event['eventid'], '')
+                    else:
+                        old_sub_events.remove(old_sub_event)
+                        print("remove illegal subevent")
```

Comment:
```suggestion
                        _logger.debug("remove illegal subevent")
```
Could we use _logger debug statement here? Also, illegal might not be the appropriate word. Are we trying to say deleted subevent?